extensions/oidc-client/deployment/pom.xml

```diff
@@ -51,6 +51,7 @@
         <dependency>
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-core</artifactId>
+            <version>${keycloak.version}</version>
```

Comment:
We can still keep this one in the bom rather than adding a version here

Reply:
Sure, I will keep it in BOM.